sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluate/_telemetry/__init__.py

```diff
@@ -16,10 +16,10 @@
 from promptflow.core import Prompty as prompty_core
 from typing_extensions import ParamSpec
 
-from azure.ai.evaluation._model_configurations import AzureAIProject
+from azure.ai.evaluation._model_configurations import AzureAIProject, EvaluationResult
```

Comment:
Since we're not enabling client-side telemetry, I wonder if we should just remove the '_telemetry' folder. Otherwise, people exploring our code might find it confusing or concerning.

Reply:
I would rather leave it in since we could enable opt-in telemetry too. How about a follow up for it?

Reply:
sgtm